index.html

```diff
@@ -201,10 +202,11 @@ <h3>Multi-Post Stories</h3>
                   </ul>
                 </li>
               </ul>
-              <button type="button" class="button">See Project</button>
+              <button type="button" id="muiltipostbtn" class="button">See Project</button>
             </div>
           </div>
         </div>
+        <div class="featuredmodal"></div>
       </section>
 
       <section class="myProjects">
```

Comment:
- [ ] Nice job :clap: on using Javascript to popup a window containing information however The requirements say that all the code concerning `myProjects` should be created using Javascript and be rendered when the page loads. This 👇  is what the requirements state
 
> The projects section is created dynamically with the information from the projects array (all the HTML of the section is created when the page loads). **This requires changes in existing code - already created HTML code for projects list should be removed and instead it should be added via JavaScript.** Check examples of students' projects:  [here](https://github.com/olawale-o/portfolio-mobile-version/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R95) and [here 👇 ](https://github.com/vladanvi99/my-web-portfolio/pull/15/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051L96.)

Reply:
Hello @andersonlebon thank you for this review. 
I have made the necessary changes